src/plat/bcm2711/overlay-rpi4-2gb.dts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2023, UNSW
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+/ {
+    /delete-node/ memory;
+
+    /* Memory for the 2GB RAM variant of the RPi4B, see overlay-pi4.dts for an
+     * explanation of the memory ranges.
+     */
+    memory@0 {
+        device_type = "memory";
+        reg = < 0x00000000 0x00000000 0x3b400000
+            0x00000000 0x40000000 0x3c000000 >;
```

Comment:
This should just be `0x00000000 0x40000000 0x40000000` as the peripherals are at 0xfc000000

Reply:
Fixed